config/crd/bases

```diff
@@ -0,0 +1 @@
+../../cmd/metrics-operator/embedded/crds/
```

Comment:
We opted to symlink this instead of generating the files twice, this decision was made [here](https://github.com/openmcp-project/control-plane-operator/pull/40#discussion_r2068836561).